style.css

```diff
@@ -1,101 +1,394 @@
-  font-style: normal;
-  font-weight: 400;
-  font-size: 55px;
-  line-height: 64px;
-  color: #ff6b00;
-}
-
-.header-subtitle {
-  display: flex;
-  font-family: Roboto, sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 24px;
-  line-height: 32px;
-  color: #fff5e1;
-}
-
-.header-text-description {
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  color: #fff;
-}
-
-.social-media {
-  margin-top: 20px;
-  top: 268px;
-  left: 40px;
-  display: flex;
-  flex-direction: row;
-  list-style: none;
-}
-
-.social-media-icon {
-  padding: 15px;
-  display: flex;
-  justify-content: center;
-  align-items: flex-start;
-}
-
-.image-correction {
-  margin-top: 15px;
-  padding: 0 10px;
-}
+body,
+h1,
+h2,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  padding-left: 0;
+  padding-right: 0;
+  display: flex;
+}
+
+nav {
+  width: 100%;
+  height: 60px;
+  background: #3c3a39;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.navbar-logo {
+  display: flex;
+  padding-left: 20px;
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 700;
+  font-family: Roboto, sans-serif;
+  color: #fff;
+  text-decoration: none;
+}
+
+.navbar-menu {
+  padding-right: 0;
+}
+
+.header-box {
+  background-size: 256px;
+  width: 100%;
+  background-color: #1c1a19;
+  display: flex;
+  background-repeat: no-repeat;
+  background-position: bottom right -110px;
+  background-image: url("./icons/header-illustration.png");
+}
+
+.header-content {
+  width: 327px;
+  margin: 122px 22px;
+}
+
+.header-title {
+  display: flex;
+  width: 327px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 55px;
+  line-height: 64px;
+  color: #ff6b00;
+}
+
+.header-subtitle {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+}
+
+.header-text-description {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.social-media {
+  margin-top: 20px;
+  top: 268px;
+  left: 40px;
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+.social-media-icon {
+  padding: 15px;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.image-correction {
+  margin-top: 15px;
+  padding: 0 10px;
+}
+
+.new-container {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.recent-work {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: grid;
+  justify-content: start;
+  margin-top: 170px;
+}
+
+.about-hr {
+  border: 1px dotted #000;
+  margin-bottom: 15px;
+}
+
+.image-p {
+  display: grid;
+  padding: 20px;
+  margin: 20px;
+}
```

Comment:
- [ ] Kindly make sure that your design match the Figma template design as much as possible (e.g. font, colors, images, tests, margins)
Your design
![Capture](https://user-images.githubusercontent.com/85095702/159115431-26f4d6a8-4385-42a5-a055-b5c03155212c.PNG)
![Capture2](https://user-images.githubusercontent.com/85095702/159115440-c3298824-874d-44bb-ae0d-e581a02ef39c.PNG)
![Capture3](https://user-images.githubusercontent.com/85095702/159115454-b265a178-1edd-47bf-aa14-fc871b601639.PNG)

Figma template
![figma2](https://user-images.githubusercontent.com/85095702/159115470-09ecebb9-3271-4566-90c7-089855122768.PNG)
![figma3](https://user-images.githubusercontent.com/85095702/159115478-51d1fc81-3547-45f8-b25d-05c135c93017.PNG)
![figma](https://user-images.githubusercontent.com/85095702/159115489-1594c141-0191-49f1-a840-901d66d17290.PNG)